packages/block-library/src/categories/index.php

```diff
@@ -31,6 +31,9 @@ function render_block_core_categories( $attributes ) {
 		$wrapper_markup           = '<div class="%1$s">%2$s</div>';
 		$items_markup             = wp_dropdown_categories( $args );
 		$type                     = 'dropdown';
+		$title                    = __( 'Categories' );
+		$label_class              = 'wp-block-categories-label';
+		$items_markup             = '<label class="' . $label_class . '" for="' . $id . '">' . $title . '</label> ' . $items_markup;
```

Comment:
I believe `$title` ought to be escaped here, perhaps via `esc_html__`

https://developer.wordpress.org/reference/functions/esc_html__/